tests/EzPlatformCoreBundle/bundle/DependencyInjection/Compiler/SessionConfigurationPassTest.php

```diff
@@ -23,6 +23,20 @@ protected function registerCompilerPass(ContainerBuilder $container): void
         $container->addCompilerPass(new SessionConfigurationPass());
     }
 
+    public function testCompilesWithoutStorageDefinitions(): void
+    {
+        $this->container->setParameter('ezplatform.session.handler_id', 'my_handler');
+        $this->container->setParameter('ezplatform.session.save_path', 'my_save_path');
+
+        $this->compile();
+
+        $this->assertContainerBuilderHasAlias('session.handler', 'my_handler');
+        $this->assertContainerBuilderHasParameter('session.save_path', 'my_save_path');
+
+        $this->assertContainerBuilderNotHasService('session.storage.native');
+        $this->assertContainerBuilderNotHasService('session.storage.php_bridge');
```

Comment:
Note for reviews: technically we might not care if those services are present, so those two assertions could be removed.

:+1: / :-1: ?

Reply:
> Note for reviewers: technically we might not care if those services are present, so those two assertions could be removed.
> 
>  /  ?

No strong feelings. Could stay to protect against merge-up mistakes, but maybe this is not that important.

Reply:
To be honest after some thought I think this should be moved before the test. The most important thing is that the compiler pass does not crash the app if the session services are not there, so what we want to have is confirmation that their absence is handled.